pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -476,9 +476,9 @@ func TestGetTaskFunc(t *testing.T) {
 				t.Error(diff)
 			}
 
-			// local cluster task and  bundle task have empty source for now. This may be changed in future.
-			if configSource != nil {
-				t.Errorf("expected configsource is nil, but got %v", configSource)
+			// local cluster task and  bundle task have empty RefSource for now. This may be changed in future.
```

Comment:
Is this mentioned in any issues/design teps?

Reply:
I should have made this clear in the comment.
- `bundle` task is meant to say the tasks that are resolved via the `taskref.bundle` approach instead of remote bundle resolver. Since that approach will be [deprecated](https://github.com/tektoncd/pipeline/issues/5514), I think it should be ok to have empty refSource. 
- `local cluster task` refers to the tasks living in the local cluster that are resolved by the localcluster resolver via `taskref.kind + name + apiversion` approach instead of remote cluster resolver.